lib/rust/parser/engine/src/lexer.rs

```diff
@@ -0,0 +1,14 @@
+#![allow(clippy::all)]
+#![allow(unknown_lints)]
+#![allow(missing_copy_implementations)]
+#![allow(missing_debug_implementations)]
+#![allow(missing_docs)]
+#![allow(trivial_casts)]
+#![allow(trivial_numeric_casts)]
+#![allow(unsafe_code)]
+#![allow(unused_import_braces)]
```

Comment:
Why are these here?

Reply:
Because this file contains the generated lexer, which I assume our code guidelines should not care about.

Reply:
So these should be put in the _generated_ code, not in the file including it.